manifest/Resource.rc

```diff
@@ -92,5 +92,4 @@ END
 
 
 /////////////////////////////////////////////////////////////////////////////
-#endif    // not APSTUDIO_INVOKED
-
+#endif    // not APSTUDIO_INVOKED
```

Comment:
Any reason for newline removal

(bikeshedding is real btw)

Reply:
idk how that happened